src/kernels/utf8.rs

```diff
@@ -75,6 +75,39 @@ pub fn add_utf8_arrays(lhs: &Utf8Array<i64>, rhs: &Utf8Array<i64>) -> DaftResult
     Ok(result)
 }
 
+#[allow(dead_code)] // TODO: Integrate and remove me!
+pub fn endswith_utf8_arrays(
+    data: &Utf8Array<i64>,
+    pattern: &Utf8Array<i64>,
```

Comment:
I would have this function take in a generic function `Fn(&str, &str) -> bool` so we can reuse it for other binary boolean functions like starts with, contains, etc

Reply:
I have another PR coming up to do `contains` - just wanted to have one kernel working end to end first before generalizing